src/index.ts

```diff
@@ -416,7 +416,7 @@ export const privateToAddress = function(privateKey: Buffer): string {
 }
 
 /**
- * Returns an Account with the given private key.
+ * Returns an account with the given private key.
  * @param {Buffer} privateKey
  * @return {Account}
  */
```

Comment:
how about rename this method: privateToAccount -> privateKeyToAccount?

Reply:
Thanks for the suggestion. For now, let me just follow the convention in the file and just add a todo instead.